### ISSUE TITLE
'temporary connection' feature lost after upgrade to 22.0.4

### ISSUE THREAD

**Issue by JerryJian** [dbeaver/dbeaver#16441]
#### System information: 
- Windows 11 x64
- DBeaver 22.0.4

#### Connection specification:
- SQLite database file
- sqlite
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  no

#### Describe the problem you're observing:

I added file association of *.db file to open with DBeaver sqlite driver by editing windows registry.

The behavior of opening sqlite db by double click on file has changed after upgrade to 22.0.4.

Before DBeaver 22.0.4, the connection is opened like a temporary connection, the text of connection name is italic.
After I restart dbeaver, the connections are removed from dbeaver. This is very helpful because I usually open a lot of sqlite db file.

But after I upgrade to 22.0.4, the connection is opened as a normal connection. I must clean the connection by deleting them manually.

#### Steps to reproduce, if exist:

Windows registry value:
sqlite3_db\shell\Open with DBeaver\command
C:\Program Files\DBeaver\dbeaver.exe -con "driver=sqlite|database=%1"

#### Expect

Can i have this feature again via settings or command line? Thanks.

**Comment by E1izabeth** [comment]
Thank you for bug report.

**Comment by E1izabeth** [comment]
We added a new connection parameter `save` to set whether new connection should be saved (`save=true`) or should be temporary (`save=false`). By default a new connection is temporary. It will be available in the next release.
You can read more here: https://github.com/dbeaver/dbeaver/wiki/Command-Line#connection-parameters

**Comment by JerryJian** [comment]
@E1izabeth Thanks very much.

**Comment by dtseiler** [comment]
I'm curious why this behavior was changed. I used to have `create=false` so that it _wouldn't_ create a new connection, just a temporary one as desired. This was a breaking change for a few of us.
IIRC, `create=true` would create and save the connection, why was a new `save` parameter necessary?

**Comment by E1izabeth** [comment]
It was impossible to create and save new connection earlier https://github.com/dbeaver/dbeaver/issues/15100

We have 3 cases which can't be covered with one boolean parameter:
- create new temporary connection (`create = true`, `save = false`)
- create new connection and save it (`create = true`, `save = true`)
- find existing connection and use it (`create = false`)

**Comment by dtseiler** [comment]
Wasn't having `create=true` how we saved connections previously? My recollection was that `create=false|connect=true` was how we could create temporary connections (which is all I need). So now we need to have `create=true|save=false|connect=true` (the first two being the defaults now as well).

The problem I have is that this update broke how it was previously being used, and did so in a patch/bugfix release. The [22.0.5 release notes](https://github.com/dbeaver/dbeaver/releases/tag/22.0.5) mention that the `save` option was added but not the change in behavior for `create` that is at the core of this issue.

Anyway, just wanted to air my grievances.
